modules/apps/expando/expando-web/src/main/java/com/liferay/expando/web/internal/info/field/reader/ExpandoInfoItemFieldReader.java

```diff
@@ -80,7 +82,10 @@ public String getName() {
 
 	@Override
 	public Object getValue(Object model) {
-		return getValue(model, LocaleUtil.getDefault());
+		ServiceContext serviceContext =
```

Comment:
@MrJohn1911 The fix does not seem right.

Reason is, this is a general api method used in many places, why it should use service context's local? is it possible there is no service context?

I believe the reason you did not get the localized value is because you are using a wrong api method. Please reproduce the issue and find the right locale should be used in client side rather than fixing it in this api.